components/sections/stats-with-tweets/index.tsx

```diff
@@ -119,13 +112,20 @@ const StatCard = ({ bottleUri, iconUri, title, value, bgClass }: StatCard) => {
   );
 };
 
-const Tweet = ({ className }: { className?: string }) => {
+const Tweet = ({
+  className,
+  username,
+  userhandle,
+  date,
+  meatDetails,
+  tweetLink,
+}: { className?: string } & Partial<(typeof tweets)[0]>) => {
   return (
     <Link
-      href={"#"}
+      href={tweetLink ?? "#"}
```

Comment:
Can we also add a `rel` prop with `external` value?

[reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#external)